### DISCUSSION TITLE
Reading Skills

### DISCUSSION THREAD

**Discussion by animafps** [aiming-taxonomy/glossary#8] [Concepts]
Current definition:
`Skills that require comprehension around where enemies are and how they are moving to predict or to better identify a target.`

**Comment by deLiGHT-03** [comment]
The ability to accurately diagnose the direction, speed, and (de)acceleration of a target.

**Reply by animafps** [reply]
Diagnose prob not the correct way but think refining it to keep my previous idea of comprehension of the target is a better way as well its more than just looking its about understanding and extrapolating that comprehension to further predict and recognize patterns. As well prob find a target from visual clutter or a lack of contrast specifically from their movement patterns or visual clues
 
i.e. "The ability to accurately comprehend the direction, speed and (de)acceleration of a target" but need to see how to put my last thoughts in to it

**Reply by deLiGHT-03** [reply]
The ability to accurately extract, comprehend and predict the direction, speed and (de)acceleration of a target with respect to its player and environmental influence?

**Reply by animafps** [reply]
yeah that sounds great prob need a bit more people to look over this before we can finalize it
